pkg/gofr/datasource/pubsub/eventhub/go.mod

```diff
@@ -0,0 +1,29 @@
+module gofr.dev/pkg/gofr/datasource/pubsub/azeventhub
+
+go 1.22.3
+
+replace gofr.dev v1.21.0 => ../../../../../../gofr
```

Comment:
instead of replacing a specific version, we should use replace irrespective of version.